admin_manual/configuration_server/occ_command.rst

```diff
@@ -1296,17 +1296,17 @@ This command syncs users stored in external backend services, such as _LDAP_, _S
 .. note:: 
    It’s also `one of the commands`_ that you should run on a regular basis to ensure that your ownCloud installation is running optimally.
 
-It’s not essential to run it regularly, unless you have a large number of users who change on a semi-regular basis.
-What’s more, the job is only necessary to pick up changes from a backend with users who don't log in regularly.
+It’s not essential to run it regularly, unless you have a large number of users who’s account properties.
```

Comment:
Just reworked that section of the update so that it, hopefully, makes more sense.